MicrosoftStoreServicesSample/Controllers/CollectionsController.cs

```diff
@@ -123,6 +135,21 @@ public async Task<ActionResult<string>> Query([FromBody] ClientCollectionsQueryR
             };
             queryRequest.Beneficiaries.Add(beneficiary);
 
+            if (!string.IsNullOrEmpty(clientRequest.sbx))
+            {
+                queryRequest.SandboxId = clientRequest.sbx;
+            }
+
+            if (clientRequest.EntitlementFilters != null &&

```

Comment:
Prefer null coalescing. 

```csharp
clientRequest.EntitlementFilters?.Count > 0
```

Reply:
I would prefer to keep it as it is for readability on this specific one.